pulp_python/app/serializers.py

```diff
@@ -1,30 +1,87 @@
+    class Meta:
+        model = models.Classifier
+        fields = ('name',)
 
-    field1 = serializers.TextField()
-    field2 = serializers.IntegerField()
-    field3 = serializers.CharField()
 
-    class Meta:
-        fields = platform.ContentSerializer.Meta.fields + ('field1', 'field2', 'field3')
-        model = models.PythonContent
+class PythonPackageContentSerializer(platform.ContentSerializer):
+    """
+    A Serializer for PythonPackageContent.
     """
 
+    filename = serializers.CharField()
+    path = serializers.CharField()
+    packagetype = serializers.CharField()
+    name = serializers.CharField()
+    version = serializers.CharField()
+    metadata_version = serializers.CharField()
+    summary = serializers.CharField(required=False, allow_blank=True)
+    description = serializers.CharField(required=False, allow_blank=True)
+    keywords = serializers.CharField(required=False, allow_blank=True)
+    home_page = serializers.URLField(required=False, allow_blank=True)
+    download_url = serializers.URLField(required=False, allow_blank=True)
+    author = serializers.CharField(required=False, allow_blank=True)
+    author_email = serializers.EmailField(required=False, allow_blank=True)
+    maintainer = serializers.CharField(required=False, allow_blank=True)
+    maintainer_email = serializers.EmailField(required=False, allow_blank=True)
+    license = serializers.CharField(required=False, allow_blank=True)
+    requires_python = serializers.CharField(required=False, allow_blank=True)
+    project_url = serializers.URLField(required=False, allow_blank=True)
+    platform = serializers.CharField(required=False, allow_blank=True)
+    supported_platform = serializers.CharField(required=False, allow_blank=True)
+    requires_dist = serializers.CharField(required=False, allow_blank=True)
+    provides_dist = serializers.CharField(required=False, allow_blank=True)
+    obsoletes_dist = serializers.CharField(required=False, allow_blank=True)
+    requires_external = serializers.CharField(required=False, allow_blank=True)
+    classifiers = ClassifierSerializer(
+        required=False,
+        many=True
+    )
+
+    def create(self, validated_data):
+        """
+        Creates a PythonPackageContent
+
+        Overriding default create() to write the classifiers nested field
+        :param validated_data:
+        :return:
+        """
+        classifiers = validated_data.pop('classifiers')
+        PythonPackageContent = models.PythonPackageContent.objects.create(**validated_data)
+        for classifier in classifiers:
+            models.Classifier.objects.create(pythonpackagecontent=PythonPackageContent,
+                                             **classifier)
+        return PythonPackageContent
+
     class Meta:
-        fields = platform.ContentSerializer.Meta.fields
-        model = models.PythonContent
+        fields = platform.ContentSerializer.Meta.fields + ('filename', 'path', 'packagetype',
```

Comment:
Just personally preference, but for a list this long, I think start it on the next line.

    fields = platform.ContentSerializer.Meta.fields + (
        'filename', 'path', 'packagetype',  ... 
        ...
     )